Doc/library/importlib.rst

```diff
@@ -420,30 +420,29 @@ ABC hierarchy::
         reloaded):
 
         - :attr:`__name__`
-            The name of the module.
+            The module's fully-qualified name.
 
         - :attr:`__file__`
-            The path to where the module data is stored (not set for built-in
-            modules).
+            The location from which the module was loaded (usually the path).
+            It is not set on all modules (e.g. built-in modules).
```

Comment:
I'd rather say something like "Some modules have no location (e.g. built-in modules).  In those cases, this attribute will not exist."

Reply:
I have updated the text with a similar meaning, but keeping the verb ‘set’ as we use it in almost every attribute.